.github/workflows/build_and_test_midstream_image.yml

```diff
@@ -31,9 +31,17 @@ jobs:
           podman --version
           cd Dockerfiles/midstream/
           podman build -t midstream_image:latest -f Dockerfile --build-arg OPERATOR_SDK_VERSION=v1.4.0
+
+      - name: "Run unit tests"
+        continue-on-error: false
+        run: |
+          mkdir ./output_logs
+          podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output midstream_image:latest python3 /unit_tests.py
+
       - name: "Run example operator-metadata through midstream ci image  --> positive test"
         continue-on-error: false
         run: |
+          rm -rf ./output_logs
           mkdir ./output_logs
           podman run -it -v $PWD:/project/operator-test-playbooks -v ./output_logs:/project/output midstream_image:latest
```

Comment:
We do not need anymore following stages.
* "Run example operator-metadata through midstream ci image  --> negative parsing test"
* "Run example operator-metadata through midstream ci image  --> negative operator-sdk validation test"
Theys should be already covered by your file and they do not check return code anyway

Reply:
Removed.

Reply:
Could you explain why why do we skill the stage "Run example operator-metadata through midstream ci image  --> positive test"

Reply:
Removed redundant test that is already present in unit tests.